.travis.yml

```diff
@@ -16,22 +16,27 @@ addons:
     - libgsl0-dev
     - gfortran
 
+# run on Ubuntu 16.04
+dist: xenial
```

Comment:
in future if we will add here some other dist or os we have to limit - `$TRAVIS_BUILD_DIR/distributions/JavaScript/trigger_docker_rebuild.sh` 
only to one os/dist

Reply:
but in the past I saw that you also compiled it on mac os

Reply:
Yes, but I've dropped it. Now Anaconda is more popular and solves most of the issues.